pyrgg/functions.py

```diff
@@ -23,6 +23,25 @@
 random_system = random
 
 
+def is_weighted(max_weight, min_weight, signed):
+    """
+    Check the graph is weighted or not.
+
+    :param max_weight: maximum weight
+    :type max_weight: int
+    :param min_weight: minimum weight
+    :type min_weight: int
+    :param signed: weight sign flag
+    :type signed: bool
+    :return: result as bool
+    """
+    if max_weight == min_weight and min_weight == 0:
+        return False
+    if max_weight == min_weight and min_weight == 1 and not signed:
```

Comment:
I can't figure out the relation between not being signed and not begin weighted, here.

Reply:
Consider a graph with `1` and `-1` weights.

Reply:
Got it :+1: